scripts/create_status_report.py

```diff
@@ -0,0 +1,231 @@
+import sys
+from datetime import datetime, timedelta
+import os
+import os.path
+
+STATUS_TARGET_NAME = 0
+STATUS_TARGET_LAST_COMLETED_DATE = 1
+STATUS_TARGET_STATUS = 2
+STATUS_TARGET_STATUS_DATE = 3
+STATUS_TARGET_DURATION = 4
+STATUS_TARGET_DEPENDANCIES = 5
+STATUS_TARGET_LOG_FOLDER = 6
+
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%S"
+
+BUFFER_OF_DAYS = 15
```

Comment:
why 15 days?

Reply:
make_profiler.db file can contains a lot of records for several years. it should be limited.